src/com/goide/inspections/GoUnusedImportDeclaration.java

```diff
@@ -85,4 +105,25 @@ public void visitElement(@NotNull PsiElement o) {
       }
     });
   }
+
+  /**
+   * Returns a map of import 'name' to import spec, where name is either the import's alias or the last component of the import's path. The
+   * returned map orders imports the same way as they occur in the file.
+   */
+  private static MultiMap<String, GoImportSpec> getImportNameMap(@NotNull GoFile file) {
```

Comment:
Removed, it caused a redeclared import to be detected in another test, can you make sure the test case is still correct?